hardware/model_helper.py

```diff
@@ -11,9 +11,9 @@
 except ImportError:
 	debugPrint("Unable to import Tensorflow! Is it installed?")
 
-def predict_emotion(data: deque) -> None:
+def predict_emotion(data: deque) -> str:
 	debugPrint("Loading model....")
-	model_path = f"{Path.cwd()}/model.h5"
+	model_path = f"{Path.cwd()}/checkpoints/model.h5"
 	cpu = False
```

Comment:
Isn't this always going to attempt to use GPU then? Whats the point of having the flag

Reply:
Whoops I meant to change that, thank you